app/cdap/components/SnackbarComponent/Components/TransitionComponent/index.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import WarningRoundedIcon from '@material-ui/icons/WarningRounded';
+import { Divider } from '@material-ui/core';
+import { useStyles } from './styles';
+
+export default function TransitionComponent(props) {
```

Comment:
What's the purpose and appearance of this component?

Reply:
It's acceptable to use the `Alert` that's in Labs in this Mui version.

Reply:
This is the component that holds the UI for the toaster message used to display success or error messages.

This is passed as prop to the MUI Snackbar component and the prop name happens to be TransitionComponent
So it is named accordingly

Reply:
We have implemented this component according to the design requirement